doc/source/development/code_style.rst

```diff
@@ -9,6 +9,13 @@ pandas code style guide
 .. contents:: Table of contents:
    :local:
 
+*pandas* follows the `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_
+standard and uses `Black <https://black.readthedocs.io/en/stable/>`_
+and `Flake8 <https://flake8.pycqa.org/en/latest/>`_ to ensure a
+consistent code format throughout the project. For details see the
+`contributing guide to pandas
+<https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#python-pep8-black>`_.
```

Comment:
Sphinx can create links to our own documentation automatically, see for example: https://github.com/pandas-dev/pandas/blame/master/doc/source/ecosystem.rst#L181 which will link to this label: https://github.com/pandas-dev/pandas/blame/master/doc/source/user_guide/options.rst#L1

So, in this case something like:
```
:ref:`contributing guide to pandas<contributing.code-formatting>`
```
Should create the link to: https://github.com/pandas-dev/pandas/blame/master/doc/source/development/contributing.rst#L650